test/e2e/image-bundle/metadata/annotations.yaml

```diff
@@ -0,0 +1,3 @@
+annotations:
+  operators.coreos.com.bundle.resources: "manifests+metadata"
```

Comment:
This is the only change I think we need. Please see my comment on Vu's PR: https://github.com/operator-framework/operator-registry/pull/97#discussion_r336463215

This should be a minor change, otherwise everything looks great!

Before I lgtm, do you want to hold this for further testing on the OLM side? Or would you rather merge and make new PRs if something comes up?

Reply:
I think it's easier to go ahead and get in. Hopefully any follow ups are pretty minor.